server/views/partials/layout.njk

```diff
@@ -25,6 +20,6 @@
   {# Run JavaScript at end of the
   <body>, to avoid blocking the initial render. #}
   <script src="/assets/govuk/all.js"></script>
-  <script src="/assets/govukFrontendInit.js"></script>
+  <script src="/assets/govuk/govukFrontendInit.js"></script>
```

Comment:
Should this change?  The file is still in [assets/js/govukFrontendInit.js](https://github.com/ministryofjustice/hmpps-tier-ui/blob/main/assets/js/govukFrontendInit.js), which is mapped to `assets/govukFrontendInit.js` here: https://github.com/ministryofjustice/hmpps-tier-ui/pull/41/files#diff-2dd04ad3a4bb6f42bd16042bea419f1c8db2fafa9d28a5742bce670f6bab577bR19.

So many levels of indirection with this front-end stuff!